project/Deps.scala

```diff
@@ -46,6 +47,8 @@ object Deps {
     "org.joda"  % "joda-convert" % "1.7"
   )
 
+  val mockito = "org.mockito" % "mockito-all" % "1.9.5"
```

Comment:
Can the test be run without introducing this dependency?

It's probably not a big deal, just curious.

Reply:
It seemed a little less ugly (my opinion) to do it with a mockito.  If the reviewers/owners want the ServersetNamer refactored to just use `zk2.addrOf` I think it'll make a lot of sense to use mockito.

Reply:
Sounds good to me. @adleong any thoughts here?